ui/appui-react/src/appui-react/frontstage/FrontstageDef.tsx

```diff
@@ -253,7 +254,12 @@ export class FrontstageDef {
   }
 
   /** Handles when the Frontstage becomes inactive */
-  protected async _onDeactivated() { }
+  protected async _onDeactivated() {
+    if (this._appDefaultToolId) {
+      IModelApp.toolAdmin.defaultToolId = this._appDefaultToolId;
+      this._appDefaultToolId = undefined;
```

Comment:
This naming is confusing. appDefaultTool seems to me to be the one defined by the app, not the one that core uses by default.

Reply:
Renamed to `_toolAdminDefaultToolId`